src/caret_analyze/runtime/path.py

```diff
@@ -207,20 +212,29 @@ def _merge_records(
             right_records.rename_columns(rename_rule)
             if left_records.columns[-1] != right_records.columns[0]:
                 raise InvalidRecordsError('left columns[-1] != right columns[0]')
-            left_records = merge(
-                left_records=left_records,
-                right_records=right_records,
-                join_left_key=left_records.columns[-1],
-                join_right_key=right_records.columns[0],
-                columns=Columns.from_str(
-                    left_records.columns + right_records.columns
-                ).column_names,
-                how='left'
-            )
+            if len(right_records.data) != 0:
+                left_records = merge(
+                    left_records=left_records,
+                    right_records=right_records,
+                    join_left_key=left_records.columns[-1],
+                    join_right_key=right_records.columns[0],
+                    columns=Columns.from_str(
+                        left_records.columns + right_records.columns
+                    ).column_names,
+                    how='left'
+                )
+            else:
+                msg = 'last_callback is None. Use last_callback = False'
+                logger.warn(msg)
```

Comment:
Also, for the log, it would be better to describe the forced change of the value from True to Flase and the reason for the change.

Reply:
Fixed in https://github.com/tier4/caret_analyze/pull/516/commits/fd7b4075cb6436400a85cc54aae572df611a388d.

Reply:
The expression `subscription -> take` is only understood by development members. It would be better to use `Explicitly take message by `User` to be understood by more users.

Reply:
The sentence contains capital letters in the middle. Please correct.

Reply:
Fixed in https://github.com/tier4/caret_analyze/pull/516/commits/c4e44c1095b4d0599936e2ca04668f281c7e4e46.

Reply:
Change uppercase to lowercase in https://github.com/tier4/caret_analyze/pull/516/commits/bf93822bc0d9198bf1e00e566230d8f6dcf15c0c.